source/text.tex

```diff
@@ -13279,12 +13284,11 @@
 
 \pnum
 The contents and meaning of the header \libheaderdef{cuchar}
-are the same as the C standard library header
-\libheader{uchar.h}, except that it
-declares the additional \tcode{mbrtoc8} and \tcode{c8rtomb} functions
-and does not declare types \keyword{char16_t} nor \keyword{char32_t}.
+are the same as the C standard library header \libheader{uchar.h},
+except that it does not declare types
```

Comment:
```suggestion
except that it does not declare the types
```

If you want to keep the "the" in [cwchar.syn] then should we add it here?

Reply:
Why yes, yes it does.

So we have every variation, "does not declare types", "does not declare a type", and "does not declare the type" (twice)

Reply:
Good point, agreed!